ome_zarr/scale.py

```diff
@@ -220,10 +245,16 @@ def _by_plane(
                         if new_stack is None:
                             zct_dims = shape_5d[:-2]
                             shape_dims = zct_dims[-stack_dims:]
-                            new_stack = np.zeros(
-                                (*shape_dims, out.shape[0], out.shape[1]),
-                                dtype=base.dtype,
-                            )
+                            if isinstance(out, dask.array.Array):
+                                new_stack = dask.array.zeros(
+                                    (*shape_dims, out.shape[0], out.shape[1]),
```

Comment:
the use of out.shape here is questionable because the result of resize was delayed

Reply:
this is now resolved because the dask chunks were set appropriately